parent/pom.xml

```diff
@@ -553,6 +718,28 @@
         <version>${version.spring-boot}</version>
       </dependency>
 
+      <!--
+           In order resolve dependency issues with wiremock and spring-boot-dependencies
+           make sure this is declared before spring boot dependencies.
+      -->
+
+      <dependency>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-bom</artifactId>
+        <version>11.0.20</version>
```

Comment:
can we use a property for this ?

Reply:
Added a property with https://github.com/camunda/zeebe/pull/17280/commits/a483ac06088f323b7eb637adf88ce4b71f8aced0